R/plots.R

```diff
@@ -318,7 +318,7 @@ plot_pc_variance <- function(obj,
 #' @param units either 'est_counts' ('scaled_reads_per_base' for gene_mode) or 'tpm'
 #' @param trans a string pointing to a function to use for the transformation.
 #' @param grouping a string from the columns of \code{sample_to_covariates} in
-#' the sleuth object for which to group and color by
+#' the sleuth object for which to group and color by. At this time, 
```

Comment:
do you remember what you were going to type here?

Reply:
Oh, whoops. This comment was originally going to say that "At this time, only one column can be used"; however, I implemented a version that can handle more than one column, as they exist in the table. You can revert this line back.